CardScan/Classes/ScanBaseViewController.swift

```diff
@@ -438,13 +413,13 @@ import Vision
         
         // we allow apps that integrate to supply their own sequence of images
         // for use in testing
-        //let image = self.testingImageDataSource?.nextImage() ?? squareCardImage
+        let squareAndFullCardImage = self.testingImageDataSource?.nextSquareAndFullImage() ?? (squareCardImage, fullCardImage)
         
         if #available(iOS 11.2, *) {
             if self.scanQrCode {
                 self.blockingQrModel(pixelBuffer: pixelBuffer)
             } else {
-                self.blockingOcrModel(squareCardImage: squareCardImage, fullCardImage: fullCardImage)
+                self.blockingOcrModel(squareCardImage: squareAndFullCardImage.0, fullCardImage: squareAndFullCardImage.1)
```

Comment:
naming individual tuple elements here doesn't seem great people would need to know that .0 is square and .1 is full. For squareAndFullCardImage why not have variables directly, something like `let (square, full) = self.testingImageDataSource...`